test/components/modal.spec.js

```diff
@@ -207,6 +238,68 @@ describe('Modal spec', () => {
     })
   })
 
+  describe('bindModalShowListener', () => {
+    it('should change location hash when `modal:show` is emitted', () => {
+      const hash = '#modal-open'
+
+      emitter.emit('modal:show')
+
+      expect(window.location.hash).to.be.equal(hash)
+    })
+  })
+
+  describe('bindHashChangeListener', () => {
+    context('when there is not a location hash', () => {
+      it('should hide modal', sinon.test(function () {
+        let spy = this.spy(instance, 'hide')
```

Comment:
@vitortalaia there are some other places in which `let` is being used instead of `const`. Do you think it is necessary to analyse each one of them and refactor as needed?